sale_adapt_price_wizard/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,2 @@
+* Beescoop - Cellule IT
```

Comment:
You can rewrite `Beescoop` to `BEES coop` which is the correct spelling.

Reply:
replaced in all the manifeststs and the rst files